db/migrate/20170510125854_rename_waiting_to_start_state_to_queuedin_miq_task.rb

```diff
@@ -0,0 +1,23 @@
+class RenameWaitingToStartStateToQueuedinMiqTask < ActiveRecord::Migration[5.0]
+  class Job < ActiveRecord::Base
+    belongs_to :miq_task, :class_name =>'RenameWaitingToStartStateToQueuedinMiqTask::MiqTask'
+    self.inheritance_column = :_type_disabled
+  end
```

Comment:
Oh, i see the stub *is* used, but the association inside are not, so they can be removed.